css/index.css

```diff
@@ -90,6 +156,80 @@ p {
     border-bottom: 1px dashed black;
 }
 
+.about-logo-mid {
+display: flex;
+flex-wrap: wrap;
+justify-content: space-evenly;
+}
+
+.about-mid-text {
+    display: flex;
+}
+
+.about-mid-text button {
+    margin-top: 10px;
+    border-radius: 9px;
+    padding: 5px 20px;
+    background: white;
+}
+
+.about-mid {
+    width: 590px;
+    margin-left: 20px;
+    margin-top: 10px;
+   border-bottom: 1px dashed black;
+}
+
+
+
+.about-mid2 {
+    width: 620px;
+    margin: 10px 0 1px 20px;
+  padding-bottom: 20px;
+    border-bottom: 1px dashed black;
+}
+
+
+.about-lower-img {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    padding-top: 40px;
+}
+
+
+
+.about-lower-text {
+    display: flex;
+}
+
+.about-lower-text button {
+    margin-top: 10px;
+    margin-bottom: 20px;
+    border-radius: 9px;
+    padding: 5px 20px;
+    background: white;
+}
+
+.about-lower {
+    display: flex;
+    flex-wrap: wrap;
+    align-content: space-around;
+    width: 590px;
+    margin-left: 20px;
+    margin-top: 10px;
+    border-bottom: 1px dashed gray;
+}
+
+.about-lower2 {
+    display: flex;
+    flex-wrap: wrap;
+    align-content: space-around;
+    width: 620px;
+    margin: 10px 0 1px 20px;
+    border-bottom: 1px dashed gray;
+}
+
```

Comment:
good job using flex where it was needed and also omitting the px when value was 0